torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -638,6 +638,36 @@ def rewrite_node(node):
                     input = g.call_function(ttnn.to_layout, args=(input, TtnnRowMajorLayout()))
                 return g.call_function(ttnn.pad, args=(input, full_pad, value))
 
+            if node.target == torch.ops.aten.convolution.default:
+                output_shape = node.meta["val"].size()
+                input_shape = args[0].meta["val"].size()
+                weight_shape = args[1].meta["val"].size()
+
+                if len(input_shape) == 4:
+                    input_nhwc = g.call_function(ttnn.permute, (args[0], (0, 2, 3, 1)))
```

Comment:
I tried converting the input tensor to tiled layout here, but it yielded the same error.
```suggestion
                    input_nhwc = g.call_function(ttnn.permute, (args[0], (0, 2, 3, 1)))
                    input_nhwc = g.call_function(ttnn.to_layout, (input_nhwc,), {"layout": TtnnTileLayout()})
```